src/pgduckdb_planner.cpp

```diff
@@ -106,7 +110,10 @@ CreatePlan(Query *query, const char *query_string, ParamListInfo bound_params) {
 
 PlannedStmt *
 DuckdbPlanNode(Query *parse, int cursor_options, ParamListInfo bound_params) {
+	auto save_nestlevel = NewGUCNestLevel();
+	SetConfigOption("search_path", "pg_catalog", PGC_USERSET, PGC_S_SESSION);
```

Comment:
Let's add some comment explaining why we're doing this. Also I realized setting search_path to the empty string is functionally equivalent to setting it to pg_catalog and it seems less confusing so let's do that instead.
```suggestion
	/* Temporarily clear search_path so that the query will contain fully qualified tables. If we don't do this tables are only fully-qualified if they are not part of the current search_path. */
	auto save_nestlevel = NewGUCNestLevel();
	SetConfigOption("search_path", "", PGC_USERSET, PGC_S_SESSION);
```